hexa/pipelines/signals.py

```diff
@@ -0,0 +1,16 @@
+from uuid import uuid4
+
+from django.db.models.signals import post_delete
+from django.dispatch import receiver
+
+from hexa.workspaces.models import WorkspaceMembership
+
+from .models import PipelineRecipient
+
+
+@receiver(post_delete, sender=WorkspaceMembership, dispatch_uid=uuid4())
```

Comment:
From what I read, the dispatch_uid is a unique string identifier. In all the examples I found, it's a human string (I guess you can use that UID after in the logs). Could you use a human string instead ?